lib/jnpr/junos/utils/sw.py

```diff
@@ -72,6 +72,11 @@ def __init__(self, dev):
             self._RE_list = [x for x in dev.facts.keys()
                              if x.startswith('version_RE')]
         self._multi_RE = bool(dev.facts.get('2RE'))
+        # Branch SRX in an SRX cluster doesn't really support multi_RE
+        # functionality for SW.
+        if (dev.facts.get('personality', '') == 'SRX_BRANCH' and
```

Comment:
This change won't send re0 to rpc call, but do we need to install for different node explicitly.

Reply:
Unfortunately, this platform/configuration does not support the ability to upgrade a node other than the currently connected node. Therefore, it must be the user's responsibility to explicitly perform the `SW.install()` on each node of the cluster.